Python/compile.c

```diff
@@ -2537,9 +2537,8 @@ compiler_class(struct compiler *c, stmt_ty s)
     /* ultimately generate code for:
          <name> = __build_class__(<func>, <name>, *<bases>, **<keywords>)
        where:
-         <func> is a function/closure created from the class body;
-            it has a single argument (__locals__) where the dict
-            (or MutableSequence) representing the locals is passed
+         <func> is a zero arg function/closure created from the class body.
+            It mutates its locals namespace to build the class.
```

Comment:
```suggestion
            It mutates its locals to build the class namespace.
```
Nit, but it builds only the class namespace, not the whole thing (that happens after)